CHANGELOG

```diff
@@ -0,0 +1,29 @@
+# ChangeLog
```

Comment:
name this file CHANGELOG.md to render it as markdown on Github or also use asciidoc for the changelog ?

Reply:
+1